plugins/store/README.md

```diff
@@ -68,9 +68,9 @@ fn main() {
 Afterwards all the plugin's APIs are available through the JavaScript guest bindings:
 
 ```typescript
-import { Store } from '@tauri-apps/plugin-store'
+import { createStore } from '@tauri-apps/plugin-store'
```

Comment:
I think the function was renamed in last commit to just `create`

Reply:
Since this creates a new store every time, I think we should document `load` instead